src/routes/search-route.jsx

```diff
@@ -1,3 +1,413 @@
+                  Sort By
+                </label>
+                <div className="flex items-center gap-2">
+                  <Controller
+                    control={control}
+                    defaultValue="false"
+                    name="sortDescending"
+                    render={({ field: { onChange, value, ref } }) => {
+                      return (
+                        <ControlledCheckbox
+                          value={value}
+                          ref={ref}
+                          onChange={(e) => {
+                            onChange(e);
+                            setCriteria({
+                              ...criteria,
+                              sortDescending: !criteria?.sortDescending,
+                            });
+                          }}
+                          label="Descending"
+                        />
+                      );
+                    }}
+                  />
+                </div>
+              </div>
+              <Controller
+                control={control}
+                defaultValue={sortingOptions[0]}
+                name="sortingCriteria"
+                render={({ field: { onChange, value, ref } }) => {
+                  const val = sortingOptions.find((c) => c.value === value);
+                  return (
+                    <Select
+                      inputRef={ref}
+                      value={val}
+                      styles={{
+                        control: (baseStyles, state) => ({
+                          ...baseStyles,
+                          borderColor: state.isFocused ? "black" : "black",
+                          padding: ".5rem",
+                          fontSize: "18px",
+                          borderWidth: "2px",
+                          borderRadius: 0,
+                        }),
+                      }}
+                      theme={(theme) => ({
+                        ...theme,
+                        padding: 0,
+                        borderWidth: 0,
+                        colors: {
+                          ...theme.colors,
+                          primary25: "#FFD913",
+                          primary: "black",
+                        },
+                      })}
+                      onChange={(val) => {
+                        onChange(val.value);
+                        setCriteria({
+                          ...criteria,
+                          sortingCriteria: val.value,
+                        });
+                      }}
+                      options={sortingOptions}
+                      isMulti={false}
+                    />
+                  );
+                }}
+              />
+            </div>
+          </div>
+        </form>
+      </main>
+      <div className="flex flex-col mt-20 gap-6 mx-8">
+        <p className="font-bold text-4xl">RESULTS</p>
+        <FindMovies criteria={criteria} />
+      </div>
+      {/*<Footer />*/}
```

Comment:
From the Figma it does look like we want the Footer at the bottom of the page - is there a reason you commented it out here?

Reply:
Good catch, I just forgot to uncomment it.